src/routes/+layout.svelte

```diff
@@ -105,7 +96,7 @@
             </div>
             <div class="terms-conditions w-full md:w-11/12 flex flex-col md:flex-row-reverse md:justify-between items-center gap-3 md:gap-0 text-sm md:text-md">
                 <button class="text-slate-500" data-testid="terms-and-condition-button"
-                        on:click={openTermsAndConditionsModal}>Termes et conditions
+                        on:click={() => goToTermsAndConditions()}>Termes et conditions
```

Comment:
out of curiosity why it is not
```suggestion
                        on:click={goToTermsAndConditions}>Termes et conditions
```

Reply:
You're right, I'm not going to pass any parameters, it's not necessary to do it like that.